src/components/SearchBox.vue

```diff
@@ -24,8 +44,7 @@
   </form>
 </template>
 
-<script>
-import algoliaComponent from '../component';
+<script>import algoliaComponent from '../component';
```

Comment:
Looks like the weird syntax is coming back here @Haroenv . Expected?

Reply:
not expected, maybe @bobylito's vim setup, should be put on the new line IMO

Reply:
I'll double check. Maybe I didn't update the packages that do the formating 🤔

Reply:
So it's not an error from any linter / formatter. I might have just removed it by mistake :/

Reply:
I'm pretty sure it's because of the linter or something.
Happens to me all the time but can't remember what the real issue was.